ocaml/testsuite/tests/typing-layouts-float64/unboxed_floats_alpha.ml

```diff
@@ -0,0 +1,118 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+*)
+
+(* This file contains various tests for float#.  It's not an expect test to make
+   sure it gets tested for native code. *)
+
+(* CR layouts v2: When unboxed literals work, change this file to use them
```

Comment:
hashing.

Reply:
I was not planning to make any of these layout polymorphic in this PR (and perhaps never, at least in the case of equality and compare).  Perhaps I misunderstood the plan?

Tests showing they don't typecheck in `basics_alpha.ml` test would be good though.

Reply:
I don't think we can stop this in the type-checker, in case something is stored in a structure. Well, I suppose we don't allow that to happen yet, but we will very soon. I guess this can be punted to 2.5 then. Could you update the CR accordingly?

Reply:
I've added tests showing these don't work on unboxed floats.  As you say, we'll need them to run containing unboxed floats, in the future.  The current functions will continue to be polymorphic over values, though, since we don't have layout polymorphism.  And I think it wouldn't be crazy to not provide additional functions for unboxed floats - don't use these operations!  But we can discuss that when we get there.